examples/sample_run_local_tools_groq_stream.py

```diff
@@ -44,7 +46,7 @@ def hello_tool(city: str):
 
 
 stream = client.chat.completions.create(
-    model="gpt-4o", messages=messages, tools=th.get_tools() + local_tools, stream=True
+    model="gpt-4o", messages=messages, tools=th.get_tools() + local_tools, stream=True # Retrieve tools installed from Toolhouse
```

Comment:
`gpt-4o` is an OpenAI model but the code uses Groq, and Groq does not serve OpenAI models. You should change this to any of the [Groq supported models](https://console.groq.com/docs/models) (suggest you use Llama 3.1 or 3.2)